Gemfile

```diff
@@ -1,117 +1,65 @@
+gem 'devise', '~> 4.6.0'
+gem 'devise-guests', '0.5.0'
+gem 'font-awesome-rails'
+gem 'httparty'
+gem 'jbuilder', '2.6.0'
+gem 'jquery-rails', '4.3.1'
+gem 'nokogiri', '~> 1.10.0'
+gem 'oga'
 gem 'rails', '~> 4.2.0'
-
-# we always need sqlite to the boundwiths database
-gem 'sqlite3', '1.3.12', platforms: :ruby
-
-group :test, :production do
-  gem 'mysql2', '~> 0.4.10'
-end
-
-gem 'jdbc-sqlite3', '3.8.11.2', platforms: :jruby
-
-gem 'activerecord-jdbcmysql-adapter', '1.3.21', platforms: :jruby
-
-# Use SCSS for stylesheets
+gem 'rsolr', '1.1.2'
 gem 'sass-rails', '5.0.6'
-
-# Use Uglifier as compressor for JavaScript assets
-gem 'uglifier', '3.0.2'
-
-# Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '4.2.1'
-
-# See https://github.com/rails/execjs#readme for more supported runtimes
+gem 'sdoc', '0.4.2', group: :doc
+gem 'sqlite3', '1.3.12', platforms: :ruby # we always need sqlite to the boundwiths database
+# TODO: switch back to official gem when PR #21 is accepted and makes it into a release
+gem 'summon', git: 'https://github.com/magibney/summon.rb', branch: 'sign-empty-param-values'
 gem 'therubyracer', '0.12.2', platforms: :ruby
+gem 'traject', '2.3.3'
+gem 'tzinfo-data', '1.2017.2' # newer passenger docker image needs tzinfo-data installed for some reason
+gem 'uglifier', '3.0.2'
 
-gem "blacklight", '6.8.0', :git => 'https://github.com/magibney/blacklight.git', :branch => 'v6.8.0-solr7-qq-feedFormat'
-
-gem "jettywrapper", '2.0.4'
-
-# newer passenger docker image needs tzinfo-data installed for some reason
-gem 'tzinfo-data', '1.2017.2'
-
-gem "solr_wrapper", '0.18.1'
-
-gem 'browserify-rails', '4.0.0'
-
-# Use jquery as the JavaScript library
-gem 'jquery-rails', '4.3.1'
-
+# investigate for removal
+gem 'blacklight_range_limit', '6.0.0'
+gem 'globalid', '0.3.7'
 gem 'jquery-color'
+gem 'unicorn' # for development?
 
-# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
-# As of Blacklight v6.3.0, enabling turbolinks corrupts render of search nav.
-# See https://github.com/projectblacklight/blacklight/issues/1562
-# gem 'turbolinks', '5.0.1'
-
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '2.6.0'
-
-# bundle exec rake doc:rails generates the API under doc/api.
-gem 'sdoc', '0.4.2', group: :doc
-
-# Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use Unicorn as the app server
-gem 'unicorn'
+# potentially removable - or are they used in indexing?
+gem 'jettywrapper', '2.0.4'
+gem 'solr_wrapper', '0.18.1'
 
+# JRuby related - can we remove?
+gem 'activerecord-jdbcmysql-adapter', '1.3.21', platforms: :jruby
+gem 'jdbc-sqlite3', '3.8.11.2', platforms: :jruby
 
+# why? unicorn gem is also installed...
+# Dockerfile builds from an image with passenger already installed.
 group :production do
   gem 'passenger', '5.1.6', require: 'phusion_passenger/rack_handler'
```

Comment:
The passenger gem here likely does nothing - we deploy with a Passenger docker image